src/phoenix/server/api/mutations/api_key_mutations.py

```diff
@@ -151,10 +151,21 @@ async def create_user_api_key(
     @strawberry.mutation(permission_classes=[HasSecret, IsAuthenticated])  # type: ignore
     async def delete_system_api_key(
         self, info: Info[Context, None], input: DeleteApiKeyInput
-    ) -> DeleteSystemApiKeyMutationPayload:
+    ) -> DeleteApiKeyMutationPayload:
         assert (token_store := info.context.token_store) is not None
         api_key_id = from_global_id_with_expected_type(
             input.id, expected_type_name=SystemApiKey.__name__
         )
         await token_store.revoke(ApiKeyId(api_key_id))
-        return DeleteSystemApiKeyMutationPayload(id=input.id, query=Query())
+        return DeleteApiKeyMutationPayload(id=input.id, query=Query())
+
+    @strawberry.mutation(permission_classes=[HasSecret, IsAuthenticated])  # type: ignore
+    async def delete_user_api_key(
+        self, info: Info[Context, None], input: DeleteApiKeyInput
+    ) -> DeleteApiKeyMutationPayload:
+        assert (token_store := info.context.token_store) is not None
+        api_key_id = from_global_id_with_expected_type(
+            input.id, expected_type_name=UserApiKey.__name__
+        )
+        await token_store.revoke(ApiKeyId(api_key_id))
```

Comment:
Does this actually delete it from the DB?

Reply:
it's tough for me to follow the inheritance but i believe so here: https://github.com/Arize-ai/phoenix/blob/auth/src/phoenix/server/jwt_store.py#L202 - this is the same logic as the system keys

Reply:
I see, interesting.